nucypher/characters/unlawful.py

```diff
@@ -47,6 +48,13 @@ def from_target_ursula(cls,
 
         TODO: This is probably a more instructive method if it takes a bytes representation instead of the entire Ursula.
         """
+        try:
+            from tests.utils.middleware import EvilMiddleWare
+        except ImportError:
+            raise DevelopmentInstallationRequired(importable_name='tests.utils.middleware.EvilMiddleWare')
+        else:
+            cls.network_middleware = EvilMiddleWare()
```

Comment:
I think without _else_ section this will be read slightly better (like in other places)

Reply:
:heavy_check_mark: